lib/callable.rb

```diff
@@ -0,0 +1,14 @@
+# This mixin is to provide the boilerplate for classes that implement a single
+# public class method of `.call` - typically used by objects that are entirely
+# to perform a business transaction.
+module Callable
+  extend ActiveSupport::Concern
+
+  included do
+    def self.call(*args, **kwargs, &block)
+      new(*args, **kwargs, &block).call
```

Comment:
Nitpick: might be better to rename the second parameter to `keyword_args` for clarity. EDIT: I see the same parameter names in `Requirements::Checker`, which I assume you copied this from.

Also there are a couple of lines that seem to have snuck into the commit message and can probably be deleted:

```
Taking inspiration from the Ruby
standard library there is an in
```

Reply:
Yeah `kwargs` seems to also be a near ubiquitous convention for generic keyword args naming so I think it's the most idiomatic choice. I don't particularly like it though and I'm very much looking forward to [brakeman gaining support](https://github.com/presidentbeef/brakeman/issues/1449) for [Ruby 2.7 shorthand syntax](https://rubystyle.guide/#arguments-forwarding) so these things can be removed.

Ooh thanks about the commit message, I'll fix that.